app/helpers/truncate-text.js

```diff
@@ -0,0 +1,19 @@
+import {helper} from '@ember/component/helper';
+
+function truncateText(params, hash) {
+  const [ value ] = params;
+  const { limit } = hash;
+  let text = '';
+
+  if (value != null && value.length > 0) {
+    text = value.substr(0, limit);
```

Comment:
If you wrap this in the if statement below, it won't run when it doesn't have to. It's fine like this, but just a general best practice and good habit for when you're running a more computationally intensive task.

Reply:
It is used as return value. If i put it inside the nested if, it will always empty.